config/account.go

```diff
@@ -21,6 +21,11 @@ const (
 	ChannelWeb   ChannelType = "web"
 )
 
+const (
+	IPv4 = 32
+	IPv6 = 128
```

Comment:
Looking at this again, you may wish to consider adding more information to describe what the numbers mean. Perhaps such as `IPv4Bits` / `IPv4BitSize` / `IPv4BitLength`?

I also missed that if you don't want to export these constants, the names `ipv4` and `ipv6` are equally good choices. Just so long as the `I` and `P` match case.

Reply:
Renamed to IPv4BitSize and IPv6BitSize. Yes, they are exported.